### PR TITLE
Mention CUBIC as an example of an alternative congestion control mechanism

### PR DESCRIPTION
**Pull request by loganaden**
At least one known implementation is using BBR:
https://cs.chromium.org/chromium/src/net/third_party/quic/core/congestion_control/bbr_sender.cc?g=0

### CODE REVIEW

**Comment by ekr** [comment]
I don't think it's helpful to specifically cite BBR here. There are a lot of alternatives.

**Comment by martinthomson** [comment]
I don't think that you could reasonably say that the authors were unaware of BBR.  You might then conclude that not mentioning BBR was intentional.

**Comment by loganaden** [comment]
@ianswett i understand better now. I can change the PR to mention Cubic instead.

**Comment by loganaden** [comment]
@martinthomson understood.

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -918,7 +918,10 @@ packet is a probe packet sent after the TLP or RTO timer expires, as described
 in {{tlp}} and {{rto}}.
 
 Implementations MAY use other congestion control algorithms, and endpoints MAY
-use different algorithms from one another. The signals QUIC provides for
+use different algorithms from one another. An example would be the
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
use other algorithms, such as Cubic {{?RFC8312}}.  The signals QUIC provides for
```

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -918,7 +918,10 @@ packet is a probe packet sent after the TLP or RTO timer expires, as described
...
+experimental BBR Congestion Control Mechanism
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
```

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -918,7 +918,10 @@ packet is a probe packet sent after the TLP or RTO timer expires, as described
...
+{{?I-D.cardwell-iccrg-bbr-congestion-control}} which minimises induced
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
```

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -918,7 +918,10 @@ packet is a probe packet sent after the TLP or RTO timer expires, as described
...
+latency along delivery path. The signals QUIC provides for
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
```

**Comment by loganaden** [comment]
@mikkelfj noted. I will work on updating the generic signal section if it goes through.

**Comment by MikeBishop** [comment]
Belated, but I don't like this change.  The original text makes two points:
- You don't have to use the congestion control mechanism we describe here
- Each peer can choose independently

The changed text says the first point twice, and drops the second one.

**Pull request merged**